app/models/dataset.rb

```diff
@@ -1,3 +1,19 @@
 class Dataset < ApplicationRecord
   belongs_to :organisation
+  has_many :datafiles
+
+  validates :title, presence: true, uniqueness: true
+  validates :summary, presence: true
```

Comment:
Before we publish, we also need to validate the presence of:
1. A licence  (licence or licence_other).
2. A frequency (from a fixed list).
3. At least one datafile